src/dbus_api/filesystem.rs

```diff
@@ -59,6 +60,11 @@ pub fn create_dbus_filesystem<'a>(dbus_context: &DbusContext,
         .emits_changed(EmitsChangedSignal::Const)
         .on_get(get_uuid);
 
+    let devnode_property = f.property::<&str, _>("Devnode", ())
+        .access(Access::Read)
+        .emits_changed(EmitsChangedSignal::False)
```

Comment:
checked with agrover - it is a safe assumption it will not change.  i will update.